### ISSUE TITLE
hardcoded urls

### ISSUE THREAD

**Issue by sandro-pasquali** [viebel/klipse#174]
Noticed that there are externally hosted libraries hardcoded into the dist/ plugin files. For example:

```
...sistentVector.EMPTY_NODE,["https://viebel.github.io/klipse/repo/js/pretty_format.js"],null),cljs.core.cst$kw$comment_DASH_str,"//"],null);
klipse.common.registry.register_mode("eval-...
```
I seem to be missing information on how to change those. Is there a build argument or something I can use? I don't want to depend on those external files, or even the responsibility to ensure a network connection for this code to run.

Great tool!

**Comment by viebel** [comment]
For the moment, it's not configurable.

I clearly understand your need.

I will make it configurable.

The solution I thought about was to add a parameter to `klipse_settings` named `static_scripts_path`.

Will it fit your need?

**Comment by sandro-pasquali** [comment]
@viebel Yes, I think the idea of configuring in this way would work for me. The "just works" thing is nice, given all the plugins and so on needed for codemirror etc.

Maybe related: a build option to build (and bundle) only the files necessary for one language (say, ruby) would be nice. If that was done could prob just inline the code above.

Regardless, thanks for making this tool.

**Comment by viebel** [comment]
Until a couple of weeks ago, all the scripts were bundled into a single one
and the user had to include by himself, the scripts for language evaluation
(opal, biwascheme....).

But I really wanted to make it easy to use and with minimal size. So I made
all the plugins loaded dynamically from github pages.


Anyway, for what purpose do you plan to use klipse? And what languages will
you need?

**Comment by sandro-pasquali** [comment]
I'll be using it for embedding executable code in docs, and the code will likely be JS (es6)

**Comment by viebel** [comment]
PR  #181  makes codemirror root and scripts root configurable in the `klipse_settings`.

```
codemirror_root: '/my-codemirror-root', // the root of codemirror files
scripts_root: '/my-scripts-root', // the root of scripts files (e.g pretty_format.js, \
opal.js ...)
```

@sandro-pasquali could you please give it a try before I merge it?

**Comment by viebel** [comment]
It's merged in master and available since version 6.1.0 https://github.com/viebel/klipse/releases
